src/Symfony/Component/Security/Http/Firewall.php

```diff
@@ -132,7 +132,7 @@ protected function callListeners(RequestEvent $event, iterable $listeners)
     /**
      * @deprecated since Symfony 4.3, use callListeners instead
```

Comment:
hello deprecated method

Reply:
going to create a PR for this.

Reply:
See #31782 for the security part. Once merged, you should be able to revert all changes from the `Security\Http` namespace here.